File: cecocloud-front/src/main/resources/src/app/shared/model/menu-companyia.ts
import { MenuEmpresa } from './menu-empresa';

export class MenuCompanyia {
	
	id: number;
	codi: string;
	nom: string;
	administracio: boolean;
	empreses: MenuEmpresa[];
	
		
	constructor(
		id: number,
		codi: string,
		nom: string,
		administracio: boolean,
		empreses: MenuEmpresa[]) {
			this.id = id;
			this.codi = codi;
			this.nom = nom;
			this.administracio = administracio;
			this.empreses = empreses;
		}
}

File: cecocloud-logic-api/src/main/java/es/limit/cecocloud/logic/api/dto/MenuCompanyia.java
/**
 * 
 */
package es.limit.cecocloud.logic.api.dto;

import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Setter;

/**
 * Informació d'una companyia.
 * 
 * @author Limit Tecnologies <limit@limit.es>
 */
@Getter @Setter
@AllArgsConstructor
public class MenuCompanyia {

	private Long id;
	private String codi;
	private String nom;
	private boolean administracio;
	
	private List<MenuEmpresa> empreses;

}
